directions/main.go

```diff
@@ -81,3 +78,17 @@ func (c *Commute) GetMapsURL() string {
 
 	return url
 }
+
+func AddressIsValid(address string) bool {
+	client, err := maps.NewClient(maps.WithAPIKey(MAPS_API_KEY))
+	utils.Check(err)
```

Comment:
@marioharper - I did that initially but Go didn't like having non declarative statements outside of a function body